amida-tech/greyscale
# Task/inba 377 task soft delete

#### What's this PR do?
Changes deleting of tasks to a soft delete

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-377

#### How should this be manually tested?
- Run the task integration test from /backend: `node_modules/.bin/mocha test/task.integration.js --bail -t 10000` to populate your tasks table with tasks.
- Run Greyscale and the Auth service to be able to perform operations
- try deleting a task using postman or something else: `http://localhost:3005/testorg/v0.2/tasks/:id`
- Look in the DB (Tasks table) to make sure that the record was marked as deleted and not removed from the DB. 
- Try this also with http://localhost:3005/testorg/v0.2/projects/:projectId/users/:userId

#### Any background context you want to provide?
- You will need to be able to log in to the auth service first to get a JWT token

#### Screenshots (if appropriate):
